source/Calamari/Deployment/PackageRetention/Repositories/JsonJournalRepository.cs

```diff
@@ -92,5 +108,20 @@ void Save()
                 fileSystem.OverwriteAndDelete(journalPath, tempFilePath);
             }
         }
+
+        static bool TryParseJournal(string json, out List<JournalEntry> result)
+        {
+            try
+            {
+                result = JsonConvert.DeserializeObject<List<JournalEntry>>(json);
+                return true;
+            }
+            catch (Exception e)
+            {
+                Log.Error($"Unable to parse the package retention journal file. Error message: {e.Message}");
```

Comment:
Given that we're telling the customer about this issue as a warning on line 84, lets make this Log.Verbose, so this info is recorded, but not shown up front.  Changing e.Message to e.ToString() will give more detail for this too, including the exception type, and a stack trace.